pkg/metric_generator/generator.go

```diff
@@ -73,6 +74,10 @@ func (g *FamilyGenerator) Generate(obj interface{}) *metric.Family {
 	family := g.GenerateFunc(obj)
 	family.Name = g.Name
 	family.Type = g.Type
+	// OpenMetrics spec requires that all Info metrics have a _info suffix.
+	if family.Type == metric.Info && !strings.HasSuffix(family.Name, "_info") {
+		klog.Warningf("Info metric %s does not have _info suffix", family.Name)
```

Comment:
Can we do structured logging here? (I believe WarningS doesn't exist and we'd need to fall back on InfoS)